lib/scarpe/para.rb

```diff
@@ -12,6 +12,11 @@ class Para < Scarpe::Widget
     }.freeze
     private_constant :SIZES
 
+    def self.inherited(subclass)
```

Comment:
IIRC the `inherited` method on widget should still apply when `Para` is inherited since it's a subclass of widget

Reply:
Yeah, that's what I thought too. But for some reason it doesn't work 🤷‍♂️ 
I see we're doing the same thing in `TextWidget` even when it's a subclass of widget:

https://github.com/Schwad/scarpe/blob/cdf15cf65faf6f98b8a8597b82e3e9455575d516/lib/scarpe/text_widget.rb#L4-L8

Reply:
Well that's weird 🤔